olavoasantos/little-apps
# Setup CLI

> Closes #2 

### 📑 Summary

This PR creates the CLI app and exposes a `node assistant` file for its usage. It also adds two new packages to the monorepo: `@little-apps/request-handler` and `@little-apps/polyfills`.



### 🎩 Testing instructions

1. Run `pnpm build` to build all packages
2. Run `node assistant` from the root
3. `Done in 0.256 ms` (or similar) should be logged

### ✅ Checks


### ℹ Additional Information
